packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php

```diff
@@ -23,20 +23,20 @@ class="flex flex-col max-h-[540px] gap-2.5 [&>*]:flex-[0] overflow-auto scroll-s
                 >
 
                     <img 
-                        :class="`min-w-[100px] max-h-[100px] rounded-xl border transparent cursor-pointer ${activeIndex === index ? 'border border-navyBlue pointer-events-none' : 'border-white'}`"
+                        :class="`min-w-[100px] max-h-[100px] rounded-xl border transparent cursor-pointer ${activeIndex === 'image'+index ? 'border border-navyBlue pointer-events-none' : 'border-white'}`"
```

Comment:
Use `'image_' + index` or `image_${index}`. Add underscore.

Reply:
The code has been modified, and String Interpolation has been used.